library/sha512.c

```diff
@@ -83,10 +80,6 @@
 #      if defined(__clang__)
 #        if __clang_major__ < 7
 #          error "A more recent Clang is required for MBEDTLS_SHA512_USE_A64_CRYPTO_*"
-#        elif __clang_major__ < 13 || \
-              (__clang_major__ == 13 && __clang_minor__ == 0 && \
-               __clang_patchlevel__ == 0)
-           /* We implement the intrinsics with inline assembler, so don't error */
```

Comment:
Have you tested that this is okay with clang compilers around versions 9, 10, 11 etc?

Reply:
Yes. I have tested. 

For assembly code, the pragma is also needed. 

FYI: I test it with https://hub.docker.com/repository/docker/yuhaoth/test-compiler/general . I put many compilers in them.